omod-1.12/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_12/OrderGroupResource1_12.java

```diff
@@ -0,0 +1,214 @@
+			description.addProperty("patient", Representation.REF);
+			description.addProperty("encounter", Representation.REF);
+			description.addProperty("orders", Representation.REF);
+			description.addProperty("orderSet", Representation.REF);
+			description.addSelfLink();
+			description.addLink("full", ".?v=" + RestConstants.REPRESENTATION_FULL);
+			return description;
+			
+		}
+		else if (rep instanceof FullRepresentation) {
+			DelegatingResourceDescription description = new DelegatingResourceDescription();
+			description.addProperty("uuid");
+			description.addProperty("display");
+			description.addProperty("voided");
+			description.addProperty("auditInfo");
+			description.addProperty("patient", Representation.DEFAULT);
+			description.addProperty("encounter", Representation.DEFAULT);
+			description.addProperty("orders", Representation.DEFAULT);
+			description.addProperty("orderSet", Representation.DEFAULT);
+			description.addSelfLink();
+			return description;
+		}
+		else if (rep instanceof RefRepresentation) {
+			DelegatingResourceDescription description = new DelegatingResourceDescription();
+			description.addProperty("uuid");
+			description.addProperty("display");
+			description.addSelfLink();
+			return description;
+		}
+		return null;
+	}
+	
+	@Override
+	public DelegatingResourceDescription getCreatableProperties() {
+		DelegatingResourceDescription description = new DelegatingResourceDescription();
+		description.addRequiredProperty("patient");
+		description.addRequiredProperty("encounter");
+		description.addProperty("orders");
+		description.addProperty("orderSet");
+		return description;
+	}
+	
+	public Model getGETModel(Representation rep) {
+		ModelImpl modelImpl = (ModelImpl) super.getGETModel(rep);
+		if (rep instanceof DefaultRepresentation || rep instanceof FullRepresentation) {
+			modelImpl
+			        .property("uuid", new StringProperty())
+			        .property("display", new StringProperty())
+			        .property("voided", new BooleanProperty())
+			        .property("patient", new RefProperty("#/definitions/PatientGetRef"))
+			        .property("encounter", new RefProperty("#/definitions/EncounterGetRef"))
+			        .property("orders", new RefProperty("#/definitions/OrderGetRef"))
+			        .property("orderSet", new RefProperty("#/definitions/OrdersetGetRef"));
+			
+		}
+		if (rep instanceof DefaultRepresentation) {
+			modelImpl
+			        .property("uuid", new StringProperty())
+			        .property("display", new StringProperty())
+			        .property("voided", new BooleanProperty())
+			        .property("patient", new RefProperty("#/definitions/PatientGetRef"))
+			        .property("encounter", new RefProperty("#/definitions/EncounterGetRef"))
+			        .property("orders", new RefProperty("#/definitions/OrderGetRef"))
+			        .property("orderSet", new RefProperty("#/definitions/OrdersetGetRef"));
+			
+		} else if (rep instanceof FullRepresentation) {
```

Comment:
Same here. Isn't this situation already covered by `if (rep instanceof DefaultRepresentation || rep instanceof FullRepresentation)` at the top ?

Reply:
you right . i have corrected it.